### PR TITLE
hello word

### PR DESCRIPTION
**Pull request by abousiyd**
helo world bla bla bla

### CODE REVIEW

File: staff/abdou-bousiyd/hello-word/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>hello-word</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>hello, word</h1>
```
Comment (abousiyd) [inline staff/abdou-bousiyd/hello-word/index.html]: bla bla bla

File: staff/abdou-bousiyd/hello-word/style.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+body {
```
Comment (abousiyd) [inline staff/abdou-bousiyd/hello-word/style.css]: bla nbla bla

Comment (FedeLacabaratz) [reply staff/abdou-bousiyd/hello-word/style.css]: Ok for me!

File: staff/abdou-bousiyd/hello-word/index.html (hunk 1 above)
Comment (FedeLacabaratz) [reply staff/abdou-bousiyd/hello-word/index.html]: Perfect!

File: staff/abdou-bousiyd/hello-word/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+    <script src="main.js"></script>
```
Comment (FedeLacabaratz) [inline staff/abdou-bousiyd/hello-word/index.html]: Hello

Comment (abousiyd) [reply staff/abdou-bousiyd/hello-word/index.html]: kkkk

File: staff/abdou-bousiyd/hello-word/index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>hello-word</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>hello, word</h1>
+    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nesciunt!</p>
```
Comment (FedeLacabaratz) [inline staff/abdou-bousiyd/hello-word/index.html]: Hey this doesn't make any sense!

Comment (abousiyd) [reply staff/abdou-bousiyd/hello-word/index.html]: now i change it, thanks

**Pull request merged**